src/clang.rs

```diff
@@ -34,39 +48,67 @@ impl Cursor {
         }
     }
 
+    /// Is this cursor's referent a declaration?
     pub fn is_declaration(&self) -> bool {
         unsafe { clang_isDeclaration(self.kind()) != 0 }
     }
 
+    /// Get the null cursor, which has no referent.
     pub fn null() -> Self {
         Cursor { x: unsafe { clang_getNullCursor() } }
     }
 
-    // common
```

Comment:
Didn't write this `common` comment, but I bet it refers that it the following methods apply to every cursor :)